src/silx/gui/colors.py

```diff
@@ -331,17 +331,19 @@ class Colormap(qt.QObject):
     """constant for autoscale using mean +/- 3*std(data)
     with a clamp on min/max of the data"""
 
-    PERCENTILE_1_99 = "percentile_1_99"
-    """constant for autoscale using 1st and 99th percentile of data"""
+    PERCENTILE = "percentile"
```

Comment:
That's an API break, we need to keep `PERCENTILE_1_99` for backward compatibility and add a new one here.